data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -26,8 +26,14 @@ for name in ironic ironic-inspector ironic-ramdisk-logs dnsmasq httpd coreos-dow
 done
 
 # Start the provisioning nic if not already started
-PROVISIONING_NIC={{.PlatformData.BareMetal.ProvisioningInterface}}
 
+#HACK
+#Search dmesg for the renamed interface for either eth0 or eth1
+#dmesg -t| grep 'renamed from eth0'
+#virtio_net virtio0 enp1s0: renamed from eth0
+
+DMSG_PROVISIONING_NIC={{.PlatformData.BareMetal.ProvisioningInterface}}
+PROVISIONING_NIC=`dmesg -t| grep 'renamed from '$DMSG_PROVISIONING_NIC | sed s/://g | awk '{print $3}'`
 {{ if .PlatformData.BareMetal.ProvisioningIP }}
```

Comment:
@bn222 is this what you had in mind?

Reply:
well, now 2 spaces got removed :). I would keep one. Thanks

Reply:
thanks for fixing it!